gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcEventRepository.java

```diff
@@ -176,79 +217,85 @@ public Optional<Event> findById(String id) throws TechnicalException {
                 "left join " +
                 EVENT_ENVIRONMENTS +
                 " ev on e.id = ev.event_id " +
+                "left join " +
+                EVENT_ORGANIZATIONS +
+                " evo on e.id = evo.event_id " +
                 "where e.id = ?";
             jdbcTemplate.query(builder, rowMapper, id);
             return rowMapper.getRows().stream().findFirst();
         } catch (final Exception ex) {
-            LOGGER.error("Failed to find event by id", ex);
+            log.error("Failed to find event by id", ex);
```

Comment:
For maintainability, exceptions should be either logged or rethrown but not both. see code style rule java:S2139.

This present in multiple places.